src/Centreon/Domain/Entity/EntityCreator.php

```diff
@@ -53,8 +53,7 @@ class EntityCreator
      */
     public static function createEntityByArray(string $className, array $data)
```

Comment:
add return type

Reply:
PHP does not allow to declare a return value as an 'object'. This method is a factory for all object entities. It is therefore not possible to specify a return value. The comment states a mixed return value.